src/harness/reference_models/ppa_model/ppa_ref_model.py

```diff
@@ -0,0 +1,145 @@
+#    Copyright 2016-2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+import multiprocessing
+
+import geojson
+import numpy as np
+from concurrent import futures
+from reference_models.geo import census_tract, utils, vincenty, nlcd
+from reference_models.propagation import wf_hybrid
+from reference_modesl.antenna import antenna
+from shapely import geometry, ops
+
+THRESHOLD_PER_10MHZ = -96
+RX_HEIGHT = 1.5
+
+census_tract_driver = census_tract.CensusTractDriver()
+nlcd_driver = nlcd.NlcdDriver()
+
+
+def _CalculateDbLossForEachPointAndGetContour(install_param, antenna_gain, latitude, longitude):
+  db_loss = np.zeros(len(latitude), dtype=np.float64)
+  for index, lat_lon in enumerate(zip(latitude, longitude)):
+    lat, lon = lat_lon
+    code = nlcd_driver.GetLandCoverCodes(lat, lon)
+    region_type = nlcd.GetRegionType(code)
+    db_loss[index] = wf_hybrid.CalcHybridPropagationLoss(install_param['latitude'],
+                                                         install_param['longitude'],
+                                                         install_param['height'],
+                                                         lat, lon, RX_HEIGHT,
+                                                         install_param['indoorDeployment'],
+                                                         region=region_type).db_loss
+
+  db_loss = 10.0 ** (db_loss / 10.0)
+  index_cond, _ = np.where(install_param['eirpCapability'] - db_loss + antenna_gain > THRESHOLD_PER_10MHZ)
+  return index_cond.shape[0] / 5
```

Comment:
I think one my previous comment was not taken into account:

>>> Note that there is an issue here: you divide an integer number by another integer number, so you are make a C-style integer division : 7/5 = 1 (this is the case because we use Python 2.7, and we dont specify from future import division on top of the file).
You should replace with a floating point division:
  index_cond.shape[0]/5.

Reply:
Thanks for catching that! I updated it with your recommendation.